rocAL_pybind/amd/rocal/plugin/pytorch.py

```diff
@@ -270,6 +270,102 @@ def __init__(self,
         super(ROCALClassificationIterator, self).__init__(pipe, tensor_layout=pipe._tensor_layout, tensor_dtype=pipe._tensor_dtype,
                                                           multiplier=pipe._multiplier, offset=pipe._offset, display=display, device=device, device_id=device_id)
 
+class ROCALAudioIterator(object):
+    """
+    ROCAL iterator for audio tasks for PyTorch
+    The Tensors that are returned by the iterator will be owned by ROCAL 
+    and would be valid until next iteration.
+    Parameters
```

Comment:
@swetha097 the parameters of this Iterator and description do not match can you please check